pkg/operator/sync.go

```diff
@@ -15,8 +15,10 @@ import (
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 
-	"github.com/openshift/machine-config-operator/lib/resourceapply"
-	"github.com/openshift/machine-config-operator/lib/resourceread"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
```

Comment:
i think this should have a clearer name and indicate that it's from libgo

Reply:
the default `resourceapply` should be from libgo going forward, so maybe have a better name for the MCO's resourceapply is what we need

Reply:
ah fair enough, my eyeballs just literally can't read mcoresourceapply clearly.

Reply:
let's change that to be more readable (I'm afraid I'm so bad at naming tho)

Reply:
mcoResourceApply?

Reply:
> the default `resourceapply` should be from libgo going forward, so maybe have a better name for the MCO's resourceapply is what we need (related to [#829 (comment)](https://github.com/openshift/machine-config-operator/pull/829#discussion_r291268678))

that was my thinking here. I'll change it to `mcoResourceApply`.

Reply:
awesome!! thanks @LorbusChris !